### PR TITLE
feat(cloud-service): add type select and sorting function to excel export

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
Goooood! 👍👍👍

**Review by WANZARGEN — APPROVED** [review]
LGTM!!

File: apps/web/src/services/asset-inventory/cloud-service/modules/CloudServiceToolbox.vue [hunk 1]
```diff
@@ -142,7 +142,11 @@ export default defineComponent<Props>({
         /* excel */
         const cloudServiceResourcesApiQueryHelper = new ApiQueryHelper()
             .setPageLimit(0).setPageStart(1)
-            .setSort('count', true);
+            .setMultiSort([
+                { key: 'provider', desc: false },
+                { key: 'cloud_service_group', desc: false },
+                { key: 'count', desc: true },
+            ]);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/cloud-service/modules/CloudServiceToolbox.vue:149]: suggestion: This part is always overridden, so it doesn't seem necessary.

Comment (yuda110) [inline apps/web/src/services/asset-inventory/cloud-service/modules/CloudServiceToolbox.vue:149]: This part is always overridden, so it doesn't seem necessary.

File: apps/web/src/services/asset-inventory/cloud-service/modules/CloudServiceToolbox.vue [hunk 2]
```diff
@@ -240,14 +240,30 @@ export default defineComponent<Props>({
             const excelItems = await getCloudServiceResources();
             const excelFieldList: Array<ExcelDataField[]> = await Promise.all(excelItems.map((d) => getExcelFields(d)));
 
+
+            const errorString = ['/', '\\', '?', '*', '[', ']'];
+            const removeErrorString = (str: string): string => {
+                let result = str;
+                errorString.forEach((d) => {
+                    result = result.replace(new RegExp(`\\${d}`, 'g'), '0');
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/cloud-service/modules/CloudServiceToolbox.vue:248]: nitpick: zero!!🤪

**Review by yuda110 — APPROVED** [review]
LGTM!! I left some comments!

**Pull request merged**